tests/ztest/error_hook/src/main.c

```diff
@@ -98,10 +98,15 @@ __no_optimization static void trigger_fault_access(void)
 __no_optimization static void trigger_fault_divide_zero(void)
 {
 	int a = 1;
+#if defined(CONFIG_CPU_CORTEX_M0PLUS) && defined(CONFIG_SOC_FAMILY_STM32)
```

Comment:
Is there a reason to limit this to STM32 ?
@hakehuang how is this test behaving on CM0+ based NXP targets ?

Reply:
see generated code :
```
   0x08000800 <+16>:	ldr	r3, [pc, #32]	; (0x8000824 <trigger_fault_divide_zero+52>)
   0x08000802 <+18>:	ldr	r3, [r3, #0]
   0x08000804 <+20>:	movs	r1, r3
   0x08000806 <+22>:	ldr	r0, [r7, #4]
   0x08000808 <+24>:	bl	0x800030c <__divsi3>
```